phpunit.xml.dist

```diff
@@ -17,6 +17,10 @@
         </testsuite>
     </testsuites>
 
+    <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
+    </listeners>
```

Comment:
we should configure the `phpunit.xml.dist` via dev-kit

Reply:
Can you provide a PR @OskarStark ?

Reply:
done